rpc/types_transaction_receipt.go

```diff
@@ -30,7 +30,7 @@ type MsgFromL1 struct {
 
 type MessageStatusResp struct {
 	// The hash of a L1 handler transaction
-	TransactionHash *felt.Felt `json:"transaction_hash"`
+	Hash *felt.Felt `json:"transaction_hash"`
```

Comment:
I like all the changes, but I'm just wondering, if changing this field here gives enough information about the field